dist/windows/installer-translations/afrikaans.nsi

```diff
@@ -20,8 +20,8 @@ LangString inst_pathlimit ${LANG_AFRIKAANS} "Disable Windows path length limit (
 LangString inst_firewallinfo ${LANG_AFRIKAANS} "Adding Windows Firewall rule"
 ;LangString inst_warning ${LANG_ENGLISH} "qBittorrent is running. Please close the application before installing."
 LangString inst_warning ${LANG_AFRIKAANS} "qBittorrent is running. Please close the application before installing."
-;LangString inst_uninstall_question ${LANG_ENGLISH} "A previous installation was detected. It will be uninstalled without deleting user settings."
-LangString inst_uninstall_question ${LANG_AFRIKAANS} "A previous installation was detected. It will be uninstalled without deleting user settings."
+;LangString inst_uninstall_question ${LANG_ENGLISH} "Previous installation will be uninstalled. User settings and torrents will remain intact."
+LangString inst_uninstall_question ${LANG_AFRIKAANS} "Previous installation will be uninstalled. User settings and torrents will remain intact."
```

Comment:
>installation will be uninstalled

Seems ugly for me. Am I alone?

Reply:
@glassez, @an0n666:  Perhaps… 

> installation will be **replaced** 

…would work better? 

Since the newer version is effectively doing that to the older version here.

Reply:
> Seems ugly for me. Am I alone?
>

Please suggest something beautiful. I’ll update it.

Reply:
> @glassez, @an0n666: Perhaps…
> 
> > installation will be **replaced**
> 
> …would work better?
> 
> Since the newer version is effectively doing that to the older version here.

I don’t think anything gets “replaced“. Whatever version was installed gets uninstalled first.

Reply:
@an0n666:  How about this one? 

> Previous **version** will be uninstalled.

Reply:
I thought about using that at first. But doesn’t previous version imply an older version? What if I’m downgrading to a previous version? Then that would create confusion again.

Reply:
Maybe this? =

> Current **version** will be uninstalled.